dima/react-movies/src/components/searchBar/searchBar.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from "react";
+import SearchFilter from "./searchFilter";
+import "./searchBar.css";
+import ErrorBoundary from "../errorBoundary";
+import * as QueryString from "query-string";
+import FilterProperty from "../../enums/FilterPropery";
+import { useLocation } from "react-router";
+import ParamsToPush from "../../interfaces/paramsToPush";
+interface SearchBarProps {
+  pushParamsOnSubmit: (urlParams: ParamsToPush) => void;
+}
+
+const SearchBar = ({ pushParamsOnSubmit }: SearchBarProps) => {
+  let location = useLocation();
+  const [searchBarValue, setSearchBarValue] = useState("");
+  const [searchBy, setSearchBy] = useState(FilterProperty.title);
+
+  useEffect(() => {
+    const setFiltersFromURL = (): void => {
+      const { search, searchBy: category } = QueryString.parse(location.search);
+      setSearchBarValue(search ? search.toString() : "");
```

Comment:
It is odd that you need a toString(). We always receive a string  from URL

Reply:
tip from visual says that we can have string or string[], so in other way linter gives an error